src/content/resources/language/evolution.md

```diff
@@ -42,6 +42,19 @@ on the Dart language GitHub repo.
 
 ## Changes in each release
 
+### Dart 3.6
+_Released 11 December 2024_
+| [Dart 3.6 announcement](https://medium.com/dartlang/announcing-dart-3-6-778dd7a80983)
+
+Dart 3.6 added the [digit separator][] underscore (`_`) to the language.
+Digit separators improve readability of long number literals.
+
+```dart
+var m = 100__000_000__000_000__000_000
```

Comment:
The original number here is too big to be represented by Dart ints. Consider making it a bit smaller so it doesn't result in an error.

```suggestion
var m = 1__000_000__000_000__000_000;
```